libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/tracing/TracingFilterConfiguration.kt

```diff
@@ -46,7 +46,7 @@ data class TracingFilterConfiguration(
 }
 
 enum class Target(open val filter: String) {
-    COMMON(""),
+    // COMMON(""),
```

Comment:
Remove it entirely?

Reply:
Yes, maybe and more cleanup could be done (for instance [this check](https://github.com/element-hq/element-x-android/blob/develop/libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/tracing/TracingFilterConfiguration.kt#L39) could be removed), but I prefer to keep it around for now.